runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerPlugin.java

```diff
@@ -188,7 +189,8 @@ public void startUp()
 	{
 		overlayManager.add(overlay);
 		overlayManager.add(minimapOverlay);
-		sharingManager.addMenuOptions();
+		if (config.showImportExport()) sharingManager.addMenuOptions();
+		if (config.showClear()) sharingManager.addClearOption();
```

Comment:
This project's convention is to avoid one-line `if`s.
```suggestion
		if (config.showImportExport())
		{
			sharingManager.addMenuOptions();
		}
		if (config.showClear())
		{
			sharingManager.addClearOption();
		}
```

Reply:
I know, sorry about that. These ones seem to have slipped through when I wrote this part.